userspace/libsinsp/container_engine/cri.h

```diff
@@ -28,6 +28,7 @@ class sinsp_threadinfo;
 #include "cgroup_limits.h"
 #include "container_engine/container_engine.h"
 #include "container_info.h"
+#include <cri.h>
```

Comment:
`cri.h` includes `cri.h`?.  Is this is a different `cri.h` or was this accidental?

Reply:
It's a different cri.h (the one from `libsinsp/`). I'll check if we can easily move the whole thing to `libsinsp/container_engine/cri.*` but ideally in a subsequent PR :)